src/bundle.rs

```diff
@@ -13,6 +13,75 @@ use core::{fmt, mem};
 use crate::archetype::TypeInfo;
 use crate::Component;
 
+/// Checks if a query is satisfied by a bundle. This is primarily useful for unit tests.
+///
+/// ```rs
+/// #[derive(hecs::Bundle)]
```

Comment:
Won't using these macros in doctests cause `cargo test` to fail when the macros aren't enabled? Maybe we should use type aliases instead.